autoload/ale/rename.vim

```diff
@@ -0,0 +1,193 @@
+let s:rename_map = {}
```

Comment:
Start the file with `Author` and `Description` lines to explain the purpose of the file and who did most of the work on it.